docs/source/workflow_inputs.rst

```diff
@@ -4608,6 +4645,164 @@ In addition, the PVSystem must be connected to an inverter that is entered as a
 
   .. [#] For homes with multiple inverters, all InverterEfficiency elements must have the same value.
 
+.. _hpxml_electric_panels:
+
+HPXML Electric Panels
+*********************
+
+A single electric panel can be entered as a ``/HPXML/Building/BuildingDetails/Systems/ElectricPanels/ElectricPanel``.
+
+**Note**: An electric panel is only used and subject to having default values applied when load calculation types are specified.
+See :ref:`hpxml_electric_panel_calculations` for more information about specifying electric panel load calculation types.
+
+  =======================================================================  =======  =========  =======================  ========  =============  ============================================
+  Element                                                                  Type     Units      Constraints              Required  Default        Notes
+  =======================================================================  =======  =========  =======================  ========  =============  ============================================
+  ``SystemIdentifier``                                                     id                                           Yes                      Unique identifier
+  ``Voltage``                                                              string   V          See [#]_                 No        240            Service voltage
+  ``MaxCurrentRating``                                                     double   A          >= 0                     No        200            Service max current rating
+  ``HeadroomSpaces``                                                       integer             >= 0                     No        See [#]_       Number of unoccupied breaker spaces
+  ``RatedTotalSpaces``                                                     integer             > 0                      No                       Total number of breaker spaces
+  ``BranchCircuits``                                                       element                                      No        See [#]_       Individual branch circuits
+  ``ServiceFeeders``                                                       element                                      No        See [#]_       Individual service feeders
+  =======================================================================  =======  =========  =======================  ========  =============  ============================================
+
+  .. [#] Voltage choices are "120" or "240".
+  .. [#] If neither HeadroomSpaces nor RatedTotalSpaces provided, the following default value representing an electric panel with 3 open breaker spaces will be used: HeadroomSpaces = 3.
```

Comment:
I'm confused why this says "If neither HeadroomSpaces nor RatedTotalSpaces provided ..." and not just "If HeadroomSpaces not provided ...". Is there a different default used when HeadroomSpaces is not provided but RatedTotalSpaces is? If so, please document it.

Reply:
See above. In that case, HeadroomSpaces is _calculated_.

Reply:
Okay, so describe it. It doesn't seem complicated. Something like: If HeadroomSpaces not provided, defaults to RatedTotalSpaces minus the sum of OccupiedSpaces for all BranchCircuits, or defaults to 3 if RatedTotalSpaces also not provided.

Reply:
And then also something like: If RatedTotalSpaces not provided, defaults to HeadroomSpaces plus the sum of OccupiedSpaces for all BranchCircuits.